x/upgrade/types/storeloader_test.go

```diff
@@ -146,8 +146,11 @@ func TestSetLoader(t *testing.T) {
 			require.NotNil(t, res.Data)
 
 			// check db is properly updated
+			if tc.setLoader != nil {
+				checkStore(t, db, upgradeHeight, tc.origStoreKey, k, nil)
```

Comment:
if the deleted store is not added in the method loadVersion(), the tc.origStoreKey will still have data ,and v will never be nil